src/composables/oxc.ts

```diff
@@ -86,12 +86,17 @@ export const useOxc = createGlobalState(async () => {
     options.value = urlState.o
   }
 
+  if (urlState?.t) {
+    activeTab.value = urlState.t
+  }
+
   editorValue.value = urlState?.c ?? PLAYGROUND_DEMO_CODE
 
   watchEffect(() => {
     const serialized = JSON.stringify({
       c: editorValue.value === PLAYGROUND_DEMO_CODE ? '' : editorValue.value,
       o: options.value,
+      t: activeTab.value === 'ast' ? '' : activeTab.value,
```

Comment:
I guess you don't need to check `activeTab.value === 'ast'` if we keep the default value. Maybe I am missing something.